core/node/api_server/src/web3/namespaces/debug.rs

```diff
@@ -173,6 +175,7 @@ impl DebugNamespace {
                 vm_permit,
                 shared_args,
                 self.state.connection_pool.clone(),
+                call_overrides,
                 tx.clone(),
```

Comment:
Dumb question: I see that `tx.common_data.fee.gas_limit.as_u64()` is used below to create a `Call`. IIUC, it won't use `call_overrides` to patch this value. Is this expected?

Reply:
good point, this issue was there even before my PR